core/core-impl/pom.xml

```diff
@@ -33,8 +33,8 @@
       <artifactId>commons-text</artifactId>
     </dependency>
     <dependency>
-      <groupId>io.quarkus</groupId>
-      <artifactId>quarkus-smallrye-context-propagation</artifactId>
```

Comment:
So Smallrye has become Quarkus-compatible by default?

Reply:
Yes

Reply:
More the opposite (Quarkus become Smallrye compatible) :)

Reply:
Aha, ok 👍 😄